.gitignore

```diff
@@ -5,3 +5,6 @@ server/_private
 server/.env
 .DS_Store
 ios/rüm.xcworkspace/xcuserdata/
+ios/Pods/Google
+ios/GoogleService-Info.plist
```

Comment:
I ignored `Pods/Google` so you could review this and just run `pod install` at build. Otherwise we'd have to check in 100 files.

The second one is because the plist has some secrets in it--forgot to tell you